kubernetes/zenko/charts/mgob/values.yaml

```diff
@@ -0,0 +1,83 @@
+# Default values for zenko.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+mgob:
+  enabled: true
+  replicaCount: 1
+  image:
+    repository: stefanprodan/mgob
+    pullPolicy: IfNotPresent
+    tag: edge
+  persistentVolume:
+    enabled: true
+    accessModes:
+      - ReadWriteOnce
+    annotations: {}
+    existingClaim: ""
+    size: 10Gi
+
+  configMap:
+    name: "mgob-config"
+    defaults:
+      target:
+        database: "metadata"
+        # params: "--authenticationDatabase admin"
+      scheduler:
+        cron: "0 5 * * *"
+        retention: 7
+        timeout: 60
+      s3:
+        url: "https://s3.amazonaws.com"
+        api: "S3v4"
+      # slack:
+      #   url: "https://hooks.slack.com/services/..."
+
+    plans:
+      metadata:
+        target:
+          port: 27017
+          # username: "change_me"
+          # password: "change_me"
+        s3:
+          bucket: "my-db-backups"
+          accessKey: "ABCDEFGHIJKLMNOPQRST"
+          secretKey: "be0da49b599b24115c7f53c92c729cbb2c8a17e5"
+        # slack:
+        #   username: mgob
+        #   # 'true' to notify only on failures
+        #   warnOnly: false
+
+service:
+  name: mgob
```

Comment:
Having a static service name different from the release is confusing and could lead to conflicts especially since this is the default. Typically if the names are to be overridden then `.Values.nameOverride` is to be used which is already integrated into your `_helpers.tpl` file. This will (should) properly template the override to all resources.

Reply:
done